File: api/v2010/message/create-send-messages-copilot/create-send-messages-copilot.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

message = @client.messages
  .create(
     body: 'Phantom Menace was clearly the best of the prequel trilogy.',
     messaging_service_sid: 'MG9752274e9e519418a7406176694466fa',
     to: '+441632960675'
   )

puts message.sid

File: api/v2010/message/create-send-messages-copilot/create-send-messages-copilot.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var message = MessageResource.Create(
            body: "Phantom Menace was clearly the best of the prequel trilogy.",
            messagingServiceSid: "MG9752274e9e519418a7406176694466fa",
            to: new Twilio.Types.PhoneNumber("+441632960675")
        );

        Console.WriteLine(message.Sid);
    }
}
